prime-router/metadata/fhir_mapping/hl7/datatypes/XON/Organization.yml

```diff
@@ -165,4 +165,27 @@ telecom:
   condition: $XTNTelecom NOT_NULL
   valueOf: datatypes/XTN/ContactPoint
   generateList: true
-  specs: $XTNTelecom *
\ No newline at end of file
+  specs: $XTNTelecom *
+
+contact:
```

Comment:
oh man, this is confusing... maybe just a comment about the FHIR inventory inconsistently maps to different values?